tests/feature/TenantPostsListTest.php

```
@@ -0,0 +1,89 @@
+        $result->assertStatus(200);
+        $result->assertSee('공개 포스트');
+        $result->assertDontSee('비공개 포스트');
+    }
+
+    public function test_isolates_other_tenant_posts(): void
+    {
+        $tenantA = fake(TenantModel::class, ['subdomain' => 'a-co']);
+        $tenantB = fake(TenantModel::class, ['subdomain' => 'b-co']);
+
+        service('tenant')->setTenant($tenantA);
+        $categoryA = fake(CategoryModel::class, ['tenant_id' => $tenantA->id]);
```

PHPStan (2.1.54)

[error] Cannot access property $id on array|object.


(property.nonObject)